crates/scheduler/CHANGELOG.md

```diff
@@ -38,6 +38,7 @@
 - Permit applets to call `debup::println()` during `init()`
 - Use `debug::exit()` board API when the applet traps
 - Use `log::panic!()` on interpreter errors
+- Change led scheduler calls to return out of bounds errors instead of trapping
```

Comment:
Same here, let's move this line at the top. Also this is under the wrong version. You need to create a minor subsection under 0.3.1-git and above the patch subsection already present. Also nit regarding the description, I would suggest something a bit more concise (the word scheduler is superfluous in the scheduler changelog) and actually reusing the same line as for the prelude:

```
- Change `led::{get,set}()` to never trap and return an error instead
```